src/gltf/Raw2Gltf.cpp

```diff
@@ -937,7 +949,7 @@ ModelData* Raw2Gltf(
       }
       for (int i = 0; i < raw.GetNodeCount(); i++) {
         const RawNode& node = raw.GetNode(i);
-        const auto nodeData = gltf->nodes.ptrs[i];
+        const auto nodeData = nodesById[node.id];;
```

Comment:
As part of our personal style, we prefer not to use auto if possible.

Reply:
the 'auto' is from the original code, not modified